pkgs/tools/misc/aoc-cli/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, stdenv
+, darwin
```

Comment:
```suggestion
, Security
```

Reply:
I suggested otherwise because of this https://github.com/NixOS/nixpkgs/pull/195874#issuecomment-1278964649, I'm not sure which one is better hear

Reply:
Lets move this discussion out of this PR but IMO if we want to remove the inherits we should shorten the attr path by eg removing darwin and frameworks.